src/System.IO.FileSystem/src/System/IO/UnixFileSystem.cs

```diff
@@ -13,6 +13,8 @@ namespace System.IO
     /// <summary>Provides an implementation of FileSystem for Unix systems.</summary>
     internal sealed partial class UnixFileSystem : FileSystem
     {
+        internal const int DefaultBufferSize = 4096;
```

Comment:
How about putting this const somewhere shared rather than duplicating it here and in File?